src/doc/rustdoc/src/unstable-features.md

```diff
@@ -135,8 +135,24 @@ Book][unstable-masked] and [its tracking issue][issue-masked].
 ## Document primitives
 
 Since primitive types are defined in the compiler, there's no place to attach documentation
-attributes. The `#[doc(primitive)]` attribute is used by the standard library to provide a way to generate
-documentation for primitive types, and requires `#![feature(doc_primitive)]` to enable.
+attributes. The `#[doc(primitive)]` attribute is used by the standard library to provide a way
+to generate documentation for primitive types, and requires `#![feature(doc_primitive)]` to enable.
+
+## Document keywords
+
+This is for rust compiler internal use only.
```

Comment:
```suggestion
This is for Rust compiler internal use only.
```

Reply:
While you're at it, can you add this sentence to the beginning of the "Document primitives" section too?

Reply:
Sure!